sdk/helper/ldaputil/config.go

```diff
@@ -18,6 +19,11 @@ import (
 // Not all fields will be used by every integration.
 func ConfigFields() map[string]*framework.FieldSchema {
 	return map[string]*framework.FieldSchema{
+		"anonymous_group_search": {
+			Type:        framework.TypeBool,
+			Default:     false,
+			Description: "Use anonymous binds when performing LDAP group searches",
```

Comment:
You'll need to add another property after `Description` called `DisplayAttrs` with the field display `Name` and default `Value` to get this field to show in the UI. [Here's an example](https://github.com/hashicorp/vault/blob/master/sdk/helper/ldaputil/config.go#L99-L102) of how to do that!

Reply:
Thank you! Will do.